docs/README.md

```diff
@@ -579,7 +579,7 @@ _WARNING_: HSTS will force your users to visit the HTTPS version of your site fo
 
 If no matching certificate is found for a given virtual host, nginx-proxy will:
 * configure nginx to use the default certificate (`default.crt` with `default.key`) and return a 500 error for HTTPS,
-* force enable HTTP; i.e. `HTTPS_METHOD` will switch to `noredirect` if it was set to `nohttp` or `redirect`.
+* if `ENABLE_HTTP_ON_MISSING_CERT=true` (default) force enable HTTP; i.e. `HTTPS_METHOD` will switch to `noredirect` if it was set to `nohttp` or `redirect`.
```

Comment:
I'm not certain formulating it this way really highlight the use case for the new environment variable.

If you're not in a hurry I can try to come up with something.